featuretools/tests/synthesis/test_deep_feature_synthesis.py

```diff
@@ -746,6 +747,37 @@ def test_makes_direct_features_through_multiple_relationships(games_es):
                 assert feature_with_name(features, f)
 
 
+def test_stacks_multioutput_features(es):
+    fm, feat = ft.dfs(entityset=es,
+                      target_entity="customers",
+                      agg_primitives=[NumUnique, NMostCommon(n=3)],
+                      trans_primitives=[],
+                      max_depth=3
+                      )
+
+    for i in range(3):
+        f = 'NUM_UNIQUE(sessions.N_MOST_COMMON(log.countrycode)[%d])' % i
+        assert feature_with_name(feat, f)
+
+
+def test_seed_multi_output_feature_stacking(es):
+    threecommon = NMostCommon(3)
+    tc = ft.Feature(es['log']['product_id'], parent_entity=es["sessions"], primitive=threecommon)
+
+    stacked = ft.Feature(tc, parent_entity=es['customers'], primitive=NumUnique)
+
+    fm, feat = ft.dfs(entityset=es,
+                      target_entity="customers",
+                      seed_features=[stacked],
```

Comment:
This test will also work if we make tc the seed feature and include NumUniqe as an agg_primitive.  DFS will stack on the seed feature to create the features we are testing for